docs/.wordlist.txt

```diff
@@ -5,11 +5,17 @@ API
 APIs
 balancer
 benchmarking
+BLK
 bogo
+boolean
 bootloader
 CFS
 cfs
 CFS's
+cgroup
```

Comment:
Is there a reason we have lowercase and uppercase cgroups?

Reply:
It's just that this spellcheck is case-sensitive, and depending if you're referring to a kernel config that have `_CGROUP` on the name or just talking about `cgroup` in general, it will detect as different spelling errors.

Reply:
ok, makes sense. thanks